src/features/SendAssets/components/SendAssetsForm.tsx

```diff
@@ -279,9 +280,13 @@ export const SendAssetsForm = ({ txConfig, onComplete, protectTxButton }: ISendF
     account: object().required(translateRaw('REQUIRED')),
     address: object()
       .required(translateRaw('REQUIRED'))
-      .test('valid', translateRaw('TO_FIELD_ERROR'), function (value) {
-        return value && value.value && isValidETHAddress(value.value);
-      })
+      .when('network', (network: Network, schema: ObjectSchema) =>
+        schema.test('valid', translateRaw('TO_FIELD_ERROR', { $network: network.name }), function (
+          value
+        ) {
+          return value && value.value && isValidAddress(value.value, network.chainId);
```

Comment:
This looks to work, but I am not sure it will be possible to send without fixing the issues in Ethers. Balance scanning etc seems to be failing on RSK right now. Have you talked to ricmoo or is there an open issue?